README.md

```diff
@@ -0,0 +1,47 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# OOP School Library
+
+> A ruby on rails app for maintaining personnel budgets.
+
+## Live Link
+[LIVE DEMO](https://budgeting-app-rails.herokuapp.com/)
+## Built With
+
+- Ruby
+- Ruby on Rails
+- RSpec
+
+## Getting Started
+
+To get a local copy follow these simple example steps.  
+
+1.- Open the terminal window and clone the repository using this command:  
+`git clone git@github.com:Saadat123456/budget-app.git` 
+
+2.- Change the directory to the school-library directory  
+`cd budget-app`  
+3.- Install the project's dependencies by running this command:   
+`bundle install`
+
+## Authors
+
+👤 **Saadat Ali**
+
+- GitHub: [@githubhandle](https://github.com/Saadat123456)
+- Twitter: [@twitterhandle](https://twitter.com/Saadat02021999)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/saadatali1999/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
```

Comment:
- [ ] your README file should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).